### PR TITLE
Update th_TH.ini V.9

### PR DESCRIPTION
**Pull request by Saramagrean**
Add+Edit Remote ISO and more. :)

### CODE REVIEW

File: th_TH.ini [hunk 1]
```diff
@@ -728,10 +725,10 @@ General = ทั่วไป
 Help the PPSSPP team = ช่วยเหลือทีมงาน PPSSPP
 Host (bugs, less lag) = โฮสต์ (อาจเกิดบั๊ก, แลคน้อยลง)
 I/O on thread (experimental) = รับ/ส่ง บนเธรด (ขั้นทดสอบ)
-Interpreter = Interpreter
+Interpreter = อินเตอร์พรีเตอร์
 IO timing method = ทางเลือกคำนวณเวลา รับ/ส่ง ข้อมูล
-iOS9NoDynarec = Dynarec (JIT) : JIT ไม่สามารถทำงานได้บน arm64 iOS 9
-IR Interpreter = IR interpreter
+IR Interpreter = ไออาร์ อินเตอร์พรีเตอร์
+iOS9NoDynarec = Dynarec (JIT) : JIT ไม่สามารถทำงานได้บน ARM64 iOS 9
```
Comment (vnctdj) [inline th_TH.ini]: Please don't break sorting order.

File: th_TH.ini [hunk 2]
```diff
@@ -500,6 +500,7 @@ Audio = เสียง
 Controls = ควบคุม
 Graphics = ภาพ
 Networking = เครือข่าย
+Remote Disc Streaming = เล่นเกมผ่านสตรีมมิ่ง
```
[the remark this answers is not in the document]
Comment (Saramagrean) [reply th_TH.ini]: OK.

File: th_TH.ini [hunk 3]
```diff
@@ -511,34 +512,33 @@ An.Up = อนาล็อกขึ้น
...
 RightAn.Left = อนาล็อก(ขวา)ซ้าย
 RightAn.Right = อนาล็อก(ขวา)ขวา
 RightAn.Up = อนาล็อก(ขวา)ขึ้น
 Save State = บันทึกสเตทเกม
-Select = Select
+Select = ซีเล็คท์
 SpeedToggle = เร่งสปีดเสริม
-Square = Square
-Start = Start
-Toggle Fullscreen = Toggle fullscreen
```
[the remark this answers is not in the document]
Comment (Saramagrean) [reply th_TH.ini]: Opp, sorry. 
It's acidentally deleted.

**Comment by Saramagrean** [comment]
Done. :)

File: th_TH.ini [hunk 4]
```diff
@@ -12,13 +12,13 @@ Sound speed hack (DOA etc.) = แฮคความเร็วเสียง (
...
 
 [Controls]
+AnalogLimiter Tip = เมื่อกดที่ขอบปุ่มอนาล็อก ตัวจำกัดอนาล็อกจะทำงาน
 Analog Axis Sensitivity = ความไวต่อการตอบสนองแกนอนาล็อก
 Analog Limiter = ตัวจำกัดอนาล็อก
 Analog Mapper High End = ปรับเปลี่ยนค่าอนาล็อก high-end (การตอบสนองของแกน)
 Analog Mapper Low End = ปรับเปลี่ยนค่าอนาล็อก low-end (สลับด้าน deadzone)
 Analog Mapper Mode = โหมดการปรับอนาล็อก
 Analog Stick = ปุ่มอนาล็อก
-AnalogLimiter Tip = เมื่อกดที่ขอบปุ่มอนาล็อก ตัวจำกัดอนาล็อกจะทำงาน
```
Comment (vnctdj) [inline th_TH.ini]: Please don't break sorting order.

File: th_TH.ini [hunk 5]
```diff
@@ -698,9 +698,6 @@ Uninstall = ถอนการติดตั้ง
 24HR = 24 ชั่วโมง
 Auto = อัตโนมัติ
 Auto Load Newest Savestate = โหลดเซฟสเตทครั้งล่าสุดให้อัตโนมัติ
-AVI Dump started. = AVI dump started
-AVI Dump stopped. = AVI dump stopped
-Cache ISO in RAM = เก็บแคช ISO เอาไว้ในแรม
```
Comment (vnctdj) [inline th_TH.ini]: There is no reason to delete these ones...

File: th_TH.ini [hunk 6]
```diff
@@ -728,10 +724,10 @@ General = ทั่วไป
 Help the PPSSPP team = ช่วยเหลือทีมงาน PPSSPP
 Host (bugs, less lag) = โฮสต์ (อาจเกิดบั๊ก, แลคน้อยลง)
 I/O on thread (experimental) = รับ/ส่ง บนเธรด (ขั้นทดสอบ)
-Interpreter = Interpreter
+Interpreter = อินเตอร์พรีเตอร์
+iOS9NoDynarec = Dynarec (JIT) : JIT ไม่สามารถทำงานได้บน ARM64 iOS 9
 IO timing method = ทางเลือกคำนวณเวลา รับ/ส่ง ข้อมูล
-iOS9NoDynarec = Dynarec (JIT) : JIT ไม่สามารถทำงานได้บน arm64 iOS 9
```
Comment (vnctdj) [inline th_TH.ini]: Please don't break sorting order.

**Comment by Saramagrean** [comment]
OK, done. :D

**Pull request merged**